### PR TITLE
chore(LGTM): add LGTM config files

### PR DESCRIPTION
**Pull request by ariporad**
@nfischer: Could you review?

### CODE REVIEW

**Comment by ariporad** [comment]
LGTM

**Comment by nfischer** [comment]
:+1:

File: .lgtm [hunk 1]
```diff
@@ -0,0 +1 @@
+pattern = "(?i):shipit:|:\\+1:|LGTM|\\+1"
```
Comment (nfischer) [inline .lgtm]: Do we need `approvals = 1`?

**Comment by nfischer** [comment]
@ariporad I just have one comment about the approvals count

File: .lgtm (hunk 1 above)
Comment (ariporad) [inline .lgtm]: Good catch. I'll fix that.

Comment (ariporad) [inline .lgtm]: @nfischer: Fixed.

**Comment by ariporad** [comment]
@nfischer: Fixed. Feel free to merge.

**Pull request merged**